test/contact-forms.spec.js

```diff
@@ -124,8 +124,17 @@ describe('contact forms', () => {
       '555-123-4567', 'false', 'eng', 'yes', 'second', 'no', 'unknown',
       ['diabetes'], '', 'notes'
     ]);
+    expect(result.errors).to.be.empty;
+  });
 
+  it('create and edit household contact', async () => {
+    let result = await harness.fillContactForm('household_contact', ...[['Head', 'female', 'over5', 'no', '20', '', '', 'sister']]);
```

Comment:
please use non-deprecated interface.  can you replace that not just here but across the whole project and documentation?